### DISCUSSION TITLE
OpenInspectorOnStartup

### DISCUSSION THREAD

**Discussion by cgimenez** [wailsapp/wails#2567] [Q&A]
Hi all ! Very first steps with Wails. I'm on macOS 10.15 and need to have devtools opened at startup.
I added 
```
Debug: options.Debug{
  OpenInspectorOnStartup: true,
},
```
to the wails.Run() options but devtools still does not show...

**Comment by cgimenez** [comment]
Guard added because of : https://github.com/wailsapp/wails/issues/2394

Anyone knows what is the lacking API call in macOS 10.15 ?

**Reply by cgimenez** [reply]
Answering to myself : for what it's worth _WKInspector is available on macOS > 10.14.4
https://github.com/WebKit/WebKit/blob/main/Source/WebKit/UIProcess/API/Cocoa/_WKInspector.h

So, that should not be a problem but I can confirm that without the guard in inspector_dev.go the app crashes when using OpenInspectorOnStartup. May be the cause is elsewhere ?

**Reply by cgimenez** [reply]
Found more, in : https://github.com/wailsapp/wails/blob/424e4fc2e8d6940e9ad0f63fd323c5967d6f777b/v2/internal/frontend/desktop/darwin/inspector_dev.go#L34
When setting wait time to `10 * NSEC_PER_SEC` 
App will not crash anymore on macOS 10.15 without guard. But devtools/inspector does not show on startup.

**Reply by leaanthony** [reply]
Thanks for looking into this. Normally cross-platform issues are fun but this is cross-version weirdness 🤯
